devon4j.properties

```diff
@@ -1,14 +1,20 @@
 category=Governance
 description=Plugin for SonarQube to validate devon4j architecture and conventions.
 homepageUrl=https://github.com/devonfw/sonar-devon4j-plugin/#sonar-devon4j-plugin
-archivedVersions=3.2.1
-publicVersions=2020.04.001
+archivedVersions=3.2.1, 2020.04.001
+publicVersions=2020.04.002
 
 defaults.mavenGroupId=com.devonfw.tools
 defaults.mavenArtifactId=sonar-devon4j-plugin
 
+2020.04.002.description=Includes a fix for a bug occurring in multi-module projects and some additional minor improvements.
+2020.04.002.sqVersions=[7.9,LATEST]
+2020.04.002.date=2020-06-26
+2020.04.002.changelogUrl=https://github.com/devonfw/sonar-devon4j-plugin/blob/master/CHANGELOG.adoc#202004002
+2020.04.002.downloadUrl=https://repo.maven.apache.org/maven2/com/devonfw/tools/sonar-devon4j-plugin/2020.04.002/sonar-devon4j-plugin-2020.04.002.jar
+
 2020.04.001.description=Includes improved issue locations and rule descriptions as well as several bugfixes. First version with this new versioning scheme.
-2020.04.001.sqVersions=[7.9,LATEST]
+2020.04.001.sqVersions=[7.9,8.2]
```

Comment:
Current version is 8.3. You really want to set the end range to that. Otherwise, 8.3 users won't be prompted to upgrade in the Marketplace.

Reply:
Thanks for the hint, I'll keep that in mind for future releases.